app/templates/admin_edituser.html

```diff
@@ -72,22 +73,81 @@ <h4><i class="icon fa fa-ban"></i> Error!</h4>
                     </div>
                 </div>
                 <div class="box-footer">
-                    <button type="submit" class="btn btn-flat btn-primary">Create User</button>
+                    <button type="submit" class="btn btn-flat btn-primary">{% if create %}Create{% else %}Update{% endif %} User</button>
                 </div>
             </form>
         </div>
+        {% if not create %}
+        <div class="box box-secondary">
+            <div class="box-header with-border">
+                <h3 class="box-title">Two Factor Authentication</h3>
+            </div>
+            <div class="box-body">
+                <p>If two factor authentication was configured and is causing problems due to a lost device or technical issue, it can be disabled here.</p>
+                <p>The user will need to reconfigure two factor authentication, to re-enable it.</p>
+                <p><strong>Beware: This could compromise security!</strong></p>
+            </div>
+            <div class="box-footer">
+                <button type="button" class="btn btn-flat btn-warning button_otp_disable" id="{{ user.username }}" {% if user.otp_secret == "" %}disabled{% endif %}>Disable Two Factor Authentication</button>
```

Comment:
There is the case that otp_secret = NULL when new user created. I think we should check with `if not user.otp_secret`.

Reply:
Well spotted. I'll fix this...